lib/puppet/util/windows/user.rb

```diff
@@ -133,7 +149,7 @@ def load_profile(user, password)
   # );
   ffi_lib :advapi32
   attach_function_private :LogonUserW,
-    [:lpwstr, :lpwstr, :lpwstr, :dword, :dword, :phandle], :win32_bool
+                          [:lpwstr, :lpwstr, :lpwstr, :dword, :dword, :phandle], :win32_bool
```

Comment:
Same with these lines, not just the block at :207, we shouldn't change formatting unless the code on that line itself is changing

Reply:
Thanks @jtappa!

Might be just as easy to start from the original code and copy in just the net new changes, rather than trying to undo the various changes all over these files.